src/workspaces/NewWorkspaceModal/NewWorkspaceWizard.tsx

```diff
@@ -0,0 +1,818 @@
+            )}
+          </IdContainer>
+        </div>
+        <IdContainer>
+          {(id) => (
+            <>
+              <FormLabel htmlFor={id}>Authorization Domain</FormLabel>
+              <p style={{ marginLeft: '0.25rem' }}>
+                Authorization Domains restrict data access to only specified individuals in a group and are intended to
+                fulfill requirements you may have for data governed by a compliance standard, such as federal
+                controlled-access data or HIPAA protected data. They follow all workspace copies and cannot be removed.
+                <p>
+                  For more details, see{' '}
+                  <Link href='https://support.terra.bio/hc/en-us/articles/360026775691' {...Utils.newTabLinkProps}>
+                    When to use an Authorization Domain
+                  </Link>
+                  .
+                </p>
+              </p>
+              {!!existingGroups.length && (
+                <div style={{ marginBottom: '0.5rem', fontSize: 12 }}>
+                  <div style={{ marginBottom: '0.2rem' }}>Inherited groups:</div>
+                  {existingGroups.join(', ')}
+                </div>
+              )}
+              <Select<string, true>
+                id={id}
+                isClearable={false}
+                isMulti
+                placeholder='Select groups'
+                isDisabled={!allGroups || !billingProjects}
+                value={groups}
+                onChange={(data) => {
+                  setGroups(_.map('value', data));
+                  setEnhancedBucketLogging(!!requireEnhancedBucketLogging || data.length > 0);
+                }}
+                options={_.difference(_.uniq(_.map('groupName', allGroups)), existingGroups).sort()}
+              />
+            </>
+          )}
+        </IdContainer>
+        {renderPolicyAndWorkspaceInfo()}
+      </>
+    );
+
+    // TODO can i do this prettier with Utils.cond?
+    const handleSecondaryButtonClick = async () => {
+      if (activeTab === 'basic') {
+        await create();
+      } else if (activeTab === 'sharing') {
+        setActiveTab('basic');
+      } else if (activeTab === 'security') {
+        setActiveTab('sharing');
+      }
+    };
+
+    const handlePrimaryButtonClick = async () => {
+      if (activeTab === 'basic') {
+        setActiveTab('sharing');
+      } else if (activeTab === 'sharing') {
+        setActiveTab('security');
+      } else if (activeTab === 'security') {
+        await create();
+      }
+    };
+
+    const buttons = (
+      <div
+        style={{
+          padding: '-1 -1 rem',
+          width: '90%',
+          position: 'absolute',
+          bottom: '1rem',
+          ...modalStyles.buttonRow,
+          display: 'flex',
+          justifyContent: 'space-between',
+        }}
+      >
+        <ButtonSecondary style={{ position: 'absolute', left: '1rem' }} onClick={onDismiss}>
+          Cancel
+        </ButtonSecondary>
```

Comment:
The cancel button is not rendering correctly for me, at least on the PR site:
![Screenshot 03-03-2025 at 12 46](https://github.com/user-attachments/assets/61693443-93a2-45b1-9566-2813bfff2313)

Reply:
Ugh it looked fine locally!!